Engine/AppManager.cpp

```diff
@@ -315,6 +315,8 @@ AppManager::loadFromArgs(const CLArgs& cl)
         std::cout << "argv[" << i << "] = " << StrUtils::utf16_to_utf8( std::wstring(_imp->commandLineArgsWide[i]) ) << std::endl;
     }
 #endif
+    // This should fix GL widgets when undocked
```

Comment:
Nice to add a comment, but it would be nice to have a link to the "why". I found this one, maybe you have a better one?

https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/issues/397#issuecomment-1026914019